webapp/graphite/settings.py

```diff
@@ -66,6 +66,7 @@
 POOL_MAX_WORKERS = 10
 
 # This settings control whether https is used to communicate between cluster members
+# The timeouts also apply to all finder/readers.
```

Comment:
This comment doesn't make sense any more.

Reply:
fixed